### ISSUE TITLE
Mac OS: ghost launch configurations

### ISSUE THREAD

**Issue by bonjourjoel** [microsoft/vscode#120596]
Hi,
Since the march update, i have ghost launch configurations on vsc. It happens on Mac Os Big Sur.
I use a configuration with a networked shared folder between windows & mac to have the same code base.
When I delete a launch configuration that i had used before, it doesn't delete from the launch pick list. It stays there. If i try to run it, it says it's missing. I can't remove them: They don't belong to the workspace or the project, they just hang there forever.

I'm not sure if you care too much about my use case, but if you could be kind enough to tell me where this information is stored, that i could delete it, because these non-exitant launch configurations start to add up. Thanks.

I attach a screenshot of the issue.
![mac-ghost-launchconfigs](https://user-images.githubusercontent.com/78134277/113638399-0ee38b00-9677-11eb-917d-69b2ecffb6fe.png)

Regards.

**Comment by isidorn** [comment]
1. If you run VS Code with `--user-data-dir SOME_PATH` do you still see them?
2. If not then you should clean your local storage to get rid of them

**Comment by bonjourjoel** [comment]
Hi, I'm afraid i am not able to run vs code with arguments on a mac. For the path containing the local storage, i deleted the 6 files inside and it didn't remove the old launch configs.

**Comment by weinand** [comment]
@bonjourjoel in order to run VS Code from the command line on macOS please follow these steps: https://code.visualstudio.com/docs/setup/mac#_launching-from-the-command-line

**Comment by bonjourjoel** [comment]
Hi again,
I could run vsc on mac with arguments thanks to your help. It turns out the ghost launch configurations were not there any more.
Based on this, I tried to delete Session storage, blob storage..., I searched the 3 Cache folders, and i grepped the jsons: I couldn't find them anywhere.
So I ended up with 'mv Code Code2' and since it worked i simply did 'rm -rf Code2'.
It's cool that this Code folder is not necessary for the app to run. Deleting it completely does the trick and vsc generates a fresh one on the fly.
Problem solved for me. Thanks for your help.
Regards,
Joel
